llvm/unittests/Object/ELFTest.cpp

```diff
@@ -77,6 +77,10 @@ TEST(ELFTest, getELFRelocationTypeNameForLoongArch) {
             getELFRelocationTypeName(EM_LOONGARCH, R_LARCH_TLS_TPREL64));
   EXPECT_EQ("R_LARCH_IRELATIVE",
             getELFRelocationTypeName(EM_LOONGARCH, R_LARCH_IRELATIVE));
+  EXPECT_EQ("R_LARCH_TLS_DESC32",
```

Comment:
You don't need to update this unittest. Currently VE/LoongArch add relocation types here but the llvm-readobj test suffices.

Reply:
Removed. Thanks.